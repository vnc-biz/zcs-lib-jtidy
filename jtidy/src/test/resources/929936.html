<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2//EN">
<html>
<head>
<title>[ # ] Escape URI</title>
</head>
<body>
<a href="test\go.html">1</a><br>
<a href="test\go<.html">2</a><br>
<a href="test\go*.html">3</a><br>
<a href="test\go .html">4</a><br>
<a href="test\goç?\ ^.html">5</a><br>
<a href="go &>.html">6</a><br>
</body>
</html>