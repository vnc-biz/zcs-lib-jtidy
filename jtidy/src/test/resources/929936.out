<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2//EN">
<html>
<head>
<title>[ # ] Escape URI</title>
</head>
<body>
<a href="test/go.html">1</a><br>
<a href="test/go%3C.html">2</a><br>
<a href="test/go*.html">3</a><br>
<a href="test/go%20.html">4</a><br>
<a href="test/go%E7?/%20^.html">5</a><br>
<a href="go%20&amp;%3E.html">6</a><br>
</body>
</html>